src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public abstract double getArea();

```

Comment:
You don't need to use here public and abstract keywords, as in Java interface declare public abstract methods by default